src/pages/components/CertificateForm.js

```diff
@@ -1,12 +1,33 @@
 import React from 'react'
 import styled from 'styled-components'
 
-import { Grid } from '@digicatapult/ui-component-library'
+import { Grid, Timeline } from '@digicatapult/ui-component-library'
+
+export default function CertificateForm(props) {
+  const [items, setItems] = React.useState(props.items || undefined)
+
+  React.useEffect(() => {
+    // This is for api to retrieve status of a certificate
+    // once updated delete [timelineItems] variable
+    if (!items) {
+      fetch('localhost:3001/some-link/{id}')
+        .then((res) => res.json())
+        .then((data) => setItems(data))
+    }
+  }, [items])
```

Comment:
for now I'd remove the effect. We need to work out how to configure this and other things

Reply:
Going point, this has been removed